1.7/usage/managing-services/creating.md

```diff
@@ -0,0 +1,222 @@
+the name before the file extension.
+For example, the icon `icon-cassandra-small.png` would have a retina-ready
+alternate image named `icon-cassandra-small@2x.png`.
+
+## Step Three: Create config.json
+
+This file declares the packages configuration properties, such as the amount of CPUs, number of instances, and allotted memory.  In step four, these properties will be injected into the` marathon.json.mustache file`.
+
+Each property can provide a default value, specify whether it's required, and provide validation (minimum and maximum values).  Users can then override specific values at installation time by passing an options file to the DC/OS CLI or by setting config values through the DC/OS [UI](https://docs.mesosphere.com/current/usage/webinterface/#universe).
+
+
+```javascript
+{
+  "type": "object",
+  "properties": {
+    "foo": {
+      "type": "object",
+      "properties": {
+        "baz": {
+          "type": "integer",
+          "description": "How many times to do baz.",
+          "minimum": 0,
+          "maximum": 16,
+          "required": false,
+          "default": 4
+        }
+      },
+      "required": ["baz"]
+    }
+  },
+  "required": ["foo"]
+}
+```
+
+## Step Four: Create marathon.json.mustache
+
+This file is a [mustache template](http://mustache.github.io/) that when rendered will create a
+[Marathon](http://github.com/mesosphere/marathon) app definition capable of running your service.
+
+Variables in the mustache template will be evaluated from a union object created by merging three objects in the
+following order:
+
+1. Defaults specified in `config.json`
+
+2. User supplied options from either the DC/OS CLI or the DC/OS UI
+
+3. The contents of `resource.json`
+
+```json
+{
+  "id": "foo",
+  "cpus": "1.0",
+  "mem": "1024",
+  "instances": "1",
+  "args": ["{{{foo.baz}}}"],
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "{{resource.assets.container.docker.foo23b1cfe8e04a}}",
+      "network": "BRIDGE",
+      "portMappings": [
+        {
+          "containerPort": 8080,
+          "hostPort": 0,
+          "servicePort": 0,
+          "protocol": "tcp"
+        }
+      ]
+    }
+  }
+}
+```
+
+For information about health checks and volumes, reference the [Marathon documentation](https://mesosphere.github.io/marathon/docs/rest-api.html).
+
+## Step Five: Install Your Package
+
+At this point, your package has all the required files for running on DC/OS.  Push all of these changes up to your fork of Universe on GitHub.
+
+The next step is to verify that your new package works as expected.  To test this you are going to install it on a DC/OS cluster.
+
+You must have DC/OS and DC/OS CLI [installed](/docs/1.7/administration/installing/).
+
+#### Verify and build package
+
+Run the following from the root of the cloned Universe repository:
+
+    $ scripts/build.sh
+
+#### Install package on your cluster
+
+1.  Add your fork as a remote repository for your cluster:
```

Comment:
These instructions for adding the repo are incorrect.

If we can point them to the readme it represents everything needed. If not, the contents of the readme should be copied here so it is accurate.

https://github.com/mesosphere/universe#universe-server